core/new-gui/src/app/workspace/component/visualization-panel/visualization-panel.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { VisualizationPanelComponent } from './visualization-panel.component';
+
+describe('VisualizationPanelComponent', () => {
+  let component: VisualizationPanelComponent;
+  let fixture: ComponentFixture<VisualizationPanelComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ VisualizationPanelComponent ]
+    })
+    .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(VisualizationPanelComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
```

Comment:
try to add a test case in each component besides the default ones. For a component, test if it shows the content correctly. It doesn't need to be too many tests for componenets, but we need to make sure it shows the basic things in the test cases